NAMESPACE

```diff
@@ -3,6 +3,8 @@
 export(available_r)
 export(ga_cachix)
 export(nix_build)
+export(renv2nix)
+export(renv_lock_r_ver)
```

Comment:
I've found it useful to be able to just check the R version used by a project to gauge roughly how old the R version is that the project uses - especially old ones have in my experience been a bit more complex to debug if you need to fix and CRAN snapshot in renv for a particular date to get all the package versions recorded for instance. If it is not an R version I've used for any other projects it lets me know I'm going to be installing another R version, which might take little longer to build. 

I implemented it before taking the `renv2nix()` function approach so don't use it for anything now but thought it might be a nice convenience function, but happy to drop it if you don't think this is the place for it.

Reply:
I’d see we keep this function private.

Reply:
addressed in [0f9de72](https://github.com/ropensci/rix/pull/349/commits/0f9de7290325be6beedde83ac6fa4c46c3487522)